spec/controllers/external_works_controller_spec.rb

```diff
@@ -12,11 +12,6 @@
     context "when the URL has an external work" do
       let!(:external_work) { create(:external_work, url: url) }
 
-      it "responds with json" do
```

Comment:
Should we be updating these tests to look for `format: :js` instead now, given that's what the format changed to in the controller?